lib/utils/movie_repository_from_json.dart

```diff
@@ -0,0 +1,38 @@
+import 'dart:convert';
+import 'package:flutter/services.dart';
+import '../../../models/genres.dart';
+import '../../../models/movie.dart';
+import '../../../utils/movie_repository.dart';
+
+class MovieRepositoryFromJson implements MovieRepository {
```

Comment:
You can rename this, is not necessary to specify that you are getting the data from Json, so the class name could be renamed to MovieRepository and the interface you can renamed to IMovieRepository

Reply:
Done